### ISSUE TITLE
Could canvasXpress respect `ggplot2::update_geom_defaults`?

### ISSUE THREAD

**Issue by davidhodge931** [neuhausi/canvasXpress#155]
`ggplot2::update_geom_defaults` enables users to set default aesthetics for when an aesthetic variable is not set.

This is super useful for typing less.

I use these functions in ggblanket with a `set_blanket` function which sets default colour, fill, alpha and linewidth. 

```
library(tidyverse)
library(palmerpenguins)
library(ggblanket)

update_geom_defaults("boxplot",
                     aes(colour = "red", 
                         fill = "blue". 
                         alpha = 0.5, 
                         linewidth = 0.66)
                     )

p <- penguins |> 
  ggplot(aes(x = species, y = body_mass_g)) +
  geom_boxplot() 

p

canvasXpress::canvasXpress(p)

```

**Comment by neuhausi** [comment]
Those properties are not in the ggplot object. Where do you store them?

**Comment by davidhodge931** [comment]
Each geom has its defaults stored in it's ggproto object. 

E.g. `ggplot2::GeomBoxplot$default_aes`

**Comment by neuhausi** [comment]
Can you please explain the following code:

```
penguins |>
  drop_na(sex) |>  
  gg_jitter(
    x = species,
    y = body_mass_g,
    col = flipper_length_mm,
    facet = sex,
    x_labels = \(x) str_sub(x, 1, 1),
    y_breaks = scales::breaks_width(1000),
    y_expand_limits = 2000,
    y_labels = scales::label_number(big.mark = " "), 
    y_transform = "log10",
    y_title = "Body mass (g)",
    col_steps = TRUE,
    col_breaks = \(x) quantile(x, seq(0, 1, 0.25)),
    col_pal = viridisLite::rocket(n = 9, direction = -1),
    facet_labels = str_to_sentence,
  )
```
Where do you store the col_breaks and col_pal in the ggplot?

**Comment by neuhausi** [comment]
Fixed in 49.1
![cX-htmlwidget-e1346e02e5ae200a2fa2-cx](https://github.com/neuhausi/canvasXpress/assets/254923/614d37b3-7edd-431a-a147-708270d405a1)
